controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/role/RoleDefinition.java

```diff
@@ -81,7 +81,9 @@
               Policy.applicationUpdate,
               Policy.applicationDelete,
               Policy.applicationOperations,
-              Policy.developmentDeployment),
+              Policy.developmentDeployment,
+              Policy.keyManagement,
```

Comment:
Deploy keys will now be granted all these permissions?

Reply:
Yes.

Reply:
Well, current deploy keys will be, for a while. Only (new) developer keys will have these privileges a few weeks from now.